### PR TITLE
Feature/17 add config for linter

### LINKED ISSUE #17: Add config for linter
Right now there is no config for any linter in this project.

Things to do:
- Choose a linter
- Create a config for that linter
- Make lint run pass without warnings (either by fixing issues in code or by silencing individual warnings in the config)
- Add GH Action task that runs this linter with that config on the project

We could use pylint or flake8. Personally, I would prefer pylint because I know it best but no hard feelings. I could gift an initial config file for pylint.

### PR DESCRIPTION
**Pull request by BoxG11**
I edited the config file at .github/workflows to ignore beauty standards since most code is auto generated.
I removed six.PY3 statements.

The pylint job now runs in 2 minutes and returns no errors.

### CODE REVIEW

File: gs_api_client/swagger/api_client.py [hunk 1]
```diff
@@ -51,7 +51,6 @@ class ApiClient(object):
     PRIMITIVE_TYPES = (float, bool, bytes, six.text_type) + six.integer_types
     NATIVE_TYPES_MAPPING = {
         'int': int,
-        'long': int if six.PY3 else long,  # noqa: F821
```
Comment (bkircher) [inline gs_api_client/swagger/api_client.py:54]: This needs to be
```
'long': int,
```

File: .github/workflows/pylint.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+         pip install -r dev-requirements.txt
+    - name: Analysing the code with pylint
+      run: |
+        pylint $(git ls-files '*.py')
```
Comment (bkircher) [inline .github/workflows/pylint.yml]: Looks good, but can you join this with the existing GH Action (the one that executes the tests)? You just need to copy out this single task 

```
    - name: Analysing the code with pylint
      run: |
        pylint $(git ls-files '*.py')
```

**Review by bkircher — CHANGES_REQUESTED** [review]
Hey, thanks @jonathanlutterbeck for working on this! Looks good to me in general. Just two changes required:

1. Can you rebase this on top of current develop branch? And bring the pylint run after running pytest in the GH Action?
2. There's just a line too much removed in gs_api_client.swagger.api_client

File: .github/workflows/pylint.yml (hunk 2 above)
Comment (BoxG11) [reply .github/workflows/pylint.yml]: I merged the develop branch into this feature branch. As far as I understand that is the rebase?
The two github actions have been compressed into one file (action) and I added the line where I deleted too much.

**Comment by bkircher** [comment]
Thanks! LGTM now

**Pull request merged**